README.md

```diff
@@ -121,11 +121,24 @@ You are able to execute command functions without any argument validation. If yo
 // or...
 import webExt from 'web-ext';
 
+// If you would like to run an extension on Firefox for Android:
+
+// Path to adb binary (optional parameter, auto-detected if missing)
+const adbBin = 'path/to/adb/binary';
+// Get an array of connected device ids (Array<string>)
+const deviceIds = await webExt.util.adb.listADBDevices(adbBin);
+const adbDevice = deviceIds[0];
+// Get an array of Firefox APKs (Array<string>) on a device
+const firefoxAPKs = await webExt.util.adb.listADBFirefoxAPKs(
+  deviceId, adbBin
+);
+const firefoxApk = firefoxAPKs[0];
+
 webExt.cmd.run({
-  // To run an extension on android, uncomment next three lines
-  // target: 'firefox-android',
-  // firefoxApk: 'firefox apk package name',
-  // adbDevice: '<deviceId>',
+  // Comment out next 3 lines to run the extension on firefox-desktop
+  target: 'firefox-android',
+  firefoxApk
+  adbDevice
```

Comment:
@ankushduacodes I'm sorry, I think I wasn't clear enough in my previous comment about the addition I was thinking of to document this, I meant to add it separately from this fragment, which I think should stay minimal as it was before this change and the new snippet for android at the end of the section, here is a diff that apply the change I had in mind on top of this version of the pull request:

https://gist.github.com/rpl/1e5c11535c686c7e5be6cfead339c865#file-pr-2116-patch-for-changes-to-readme-patch

Reply:
@rpl I am sorry for not understanding prior comment correctly. I have updated the readme with the changes suggests and some additional aesthetic changes.
Please let me know if how it's looking after the change.